packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
-                // subscriptionRef.current?.cancel();
-                // subscriptionRef.current = null;
-                // setSub(null);
+                },
+                historical: config.historical,
+            });
+
+            config.processInitialData(initialData);
+            return subscription;
+        }, [query]);
+
+        useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: config.getErrorPrefix

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)